### ISSUE TITLE
[HW3] Response 302

### ISSUE THREAD

**Issue by pptnz** [swsnu/swppfall2017#70]

**Comment by LastOne817** [comment]
Yes, you can. Although I recommend using 4xx response code if you can.

p.s. `@login_required` redirects client to the login page if the client is not logged in. That's why the response code is 302. To integrate this with Angular(of course not in this hw, but your project), your frontend code should handle this response and actually redirect to the signin page.
